examples/deployment/kubernetes/README.md

```diff
@@ -0,0 +1,69 @@
+Deploying onto Kubernetes in Google Cloud
+=========================================
+
+This document guides you through the process of spinning up an example Trillian
+deployment on Google Cloud using Kubernetes and Cloud Spanner.
+
+Prerequisites
+-------------
+
+1. You should have this repo checked out :)
+1. A recent debian distribution (other platforms may work, but YMMV)
+1. You must have the `jq` binary installed
```

Comment:
because...?

E.g.
"You must have the [`jq` binary](https://packages.debian.org/stretch/jq) (for command-line manipulation of JSON) installed"

Reply:
Done